src/main/java/generator/RandomLottoGenerator.java

```diff
@@ -0,0 +1,28 @@
+package generator;
+
+import model.Lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class RandomLottoGenerator implements LottoGenerator {
+
+    public static final int VALID_MIN_LOTTO_NUMBER = 1;
+    public static final int VALID_MAX_LOTTO_NUMBER = 45;
+    public static final int LOTTO_NUMBERS_COUNT = 6;
+
+    public Lotto generateLotto() {
+        List<Integer> numbers = new ArrayList<>();
+
+        for (int i = VALID_MIN_LOTTO_NUMBER; i <= VALID_MAX_LOTTO_NUMBER; i++) {
+            numbers.add(i);
+        }
```

Comment:
캐싱해둔 `LottoNumber` 집합을 가지고 와서 아래와 같이 변경한다면 불필요한 1~45까지의 Integer 값을 가진 List를 생성할 필요가 없어서 조금 더 효율적인 코드가될 것 같아요. 😃

```suggestion
        List<LottoNumber> numbers = new ArrayList<>(LottoNumber.CACHE);
```

Reply:
수동 로또 생성 역할도 LottoGenerator에 포함시키면서 RandomLottoGenerator가 더이상 불필요하다 생각하여 클래스를 제거했습니다. [7b59b24](https://github.com/next-step/java-lotto-kakao/pull/36/commits/7b59b24f540470cfbd7f31cc55a381031323e139)